pagopa/pn-bff
# Updated SERCQ SEND address value - Feat/pn 12535

## Short description

Updated SERCQ SEND address value to `x-pagopa-pn-sercq:send-self:notification-already-delivered`

## List of changes proposed in this pull request

- Changed value into ChannelTypeMapper

## How to test

Run bff -> open postman -> call the address list api and check that the SERCQ address is returned (user colombo)
